project.mml

```diff
@@ -133,7 +133,7 @@ Layer:
               OR tourism IN ('camp_site', 'caravan_site', 'picnic_site')
               OR highway IN ('services', 'rest_area')
               OR railway = 'station')
-              AND way_area > 1*!pixel_width!::real*!pixel_height!::real
+              AND way_area > 1*POW(!scale_denominator!*0.001*0.28,2)
```

Comment:
Let's leave this line the same. If rendering 3x retina tiles, there will be 9 real pixels per standard pixel, and print quality could be even higher. At these higher resolutions the smaller polygons will be more significant and should not be excluded.